src/js/themes/base.js

```diff
@@ -693,6 +693,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       icons: {
         // color: { dark: undefined, light: undefined },
+        margin: { horizontal: 'small' },
```

Comment:
would that be a breaking change?

Reply:
don't think so. the default is current value.